websubhub-native/src/main/java/io/ballerina/stdlib/websubhub/HubCallback.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.websubhub;
+
+import io.ballerina.runtime.api.Future;
+import io.ballerina.runtime.api.Module;
+import io.ballerina.runtime.api.async.Callback;
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.types.Type;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BString;
+
+import static io.ballerina.runtime.api.utils.StringUtils.fromString;
+
+/**
+ * {@code HubCallback} used to handle the websubhub remote method invocation results.
+ */
+public class HubCallback implements Callback {
+    private final Future future;
+    private final Module module;
+
+    public HubCallback(Future future, Module module) {
+        this.future = future;
+        this.module = module;
+    }
+
+    @Override
+    public void notifySuccess(Object result) {
+        if (result instanceof BError) {
+            BError error = (BError) result;
+            if (!isModuleDefinedError(error)) {
+                error.printStackTrace();
+            }
+        }
+
+        future.complete(result);
+    }
+
+    @Override
+    public void notifyFailure(BError bError) {
+        BString errorMessage = fromString("service method invocation failed: " + bError.getErrorMessage());
+        BError invocationError = ErrorCreator.createError(module, "ServiceExecutionError",
+                errorMessage, bError, null);
+        future.complete(invocationError);
+    }
+
+    private boolean isModuleDefinedError(BError error) {
+        Type errorType = error.getType();
+        String errorName = errorType.getName();
+        Module packageDetails = errorType.getPackage();
+        String orgName = packageDetails.getOrg();
+        String packageName = packageDetails.getName();
+        return Constants.MODULE_DEFINED_ERRORS.contains(errorName)
```

Comment:
Won't the `PACKAGE_NAME` and `PACKAGE_ORG` be enough to filter module errors as only the error types reach here?